.github/workflows/leancode_app_rating-publish.yml

```diff
@@ -0,0 +1,38 @@
+name: leancode_app_rating publish
+
+on:
+  push:
+    tags: ['leancode_app_rating-v*']
+
+jobs:
+  publish:
+    name: Publish to pub.dev
+
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+
+    defaults:
+      run:
+        working-directory: mobile
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Dart
+        uses: dart-lang/setup-dart@v1
+        with:
+          sdk: 3.3
```

Comment:
Let's use 3.5.3 now. Doesn't matter much but why not use latest (for the Flutter version)